connector-splunk/src/main/java/com/redhat/cloud/notifications/connector/splunk/SplunkTrustAllManager.java

```diff
@@ -0,0 +1,26 @@
+package com.redhat.cloud.notifications.connector.splunk;
+
+import org.apache.camel.support.jsse.SSLContextParameters;
+import org.apache.camel.support.jsse.TrustManagersParameters;
+
+import javax.net.ssl.X509TrustManager;
+import java.security.cert.X509Certificate;
+
+public class SplunkTrustAllManager implements X509TrustManager {
```

Comment:
Inspired from [SplunkTrustAllCACerts.java](https://github.com/RedHatInsights/eventing-integrations/blob/3faa2fb8c58002ce5473166ade7181ece4fb14ed/splunk-quarkus/src/main/java/com/redhat/console/integrations/splunk/SplunkTrustAllCACerts.java) and [SplunkIntegration#L146-L153](https://github.com/RedHatInsights/eventing-integrations/blob/3faa2fb8c58002ce5473166ade7181ece4fb14ed/splunk-quarkus/src/main/java/com/redhat/console/integrations/splunk/SplunkIntegration.java#L146-L153).